src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500&display=swap");
+
 body {
   margin: 0;
 }
+
+h2 {
+  margin-top: 40px;
+  margin-bottom: 4px;
+  width: 166px;
+  color: var(--Main-accent, #060b35);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px; /* 150% */
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.content {
+  margin: 33px 17px 17px;
+
+  &__code {
+    color: var(--Secondary, #616070);
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+  }
+}
+
+.computer {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price__word {
+  color: var(--Secondary, #616070);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px; /* 150% */
+}
+
+.price__number {
+  color: var(--Main-accent, #060b35);
+  text-align: right;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px; /* 112.5% */
+}
+
+.stars {
+  display: flex;
+  margin-top: 16px;
+
+  &__number {
+    display: flex;
+    width: 53px;
+    height: 16px;
+    flex-direction: column;
+    justify-content: flex-end;
+    flex-shrink: 0;
+
+    color: var(--Main-accent, #060b35);
+    text-align: right;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
+  }
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
This part of the task is incomplete
<img width="546" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/93398388-5de0-4af2-9f7e-9317629761a9">